src/dbus/instropect.xml

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<!-- wpan name will vary -->
+<node name="/org/otbr/wpan%s">
+  <interface name="org.Scan">
+    <method name="Scan">
+      <!-- ret name=scan_result type=a{r{ta{b}ta{b}qqbbbbb}} -->
+    </method>
+
+    <method name="Attach">
+      <arg name="network_name" type="s"/>
```

Comment:
We want to make this a formal API definition. To reduce confusion, I'd like to keep the same names across all APIs. I think names used in [OpenThread CLI](https://github.com/openthread/openthread/blob/master/src/cli/README.md) are good candidates, so I tend to use `networkname` here.  What do you think? @jwhui @librasungirl ?